packages/components/src/templates/supernumber.stories.ts

```diff
@@ -1,6 +1,10 @@
 import '../solid-components';
 import { html } from 'lit-html';
 
+/**
```

Comment:
was this intentional? If yes, what is it used for?

Reply:
@karlbaumhauer Storybook duplicates the default story and we didn't want that behaviour. We have a solution that allows us to hide all default stories in a page, and this is a workaround that prevents the one we want from becoming invisible too.